pkg/network/tracer/connection/kprobe/tracer.go

```diff
@@ -111,8 +111,19 @@ func ClassificationSupported(config *config.Config) bool {
 	return currentKernelVersion >= classificationMinimumKernel
 }
 
+// FailedConnectionsSupported returns true if the config & TCP v4 || v6 is enabled
+func FailedConnectionsSupported(c *config.Config) bool {
```

Comment:
You could add this as a method on `config.Config` itself.

Reply:
updated